engines/director/detection_tables.h

```diff
@@ -718,7 +718,7 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "secretwriters",		"Secret Writer's Society" },
 
 	// Shareware and freeware
-	{ "101pet",				"Dalmation Adoption" },
+	{ "101pet",				"Dalmatian Adoption" },
```

Comment:
![image](https://github.com/user-attachments/assets/66b425fd-a09c-46ca-88d3-41db834a1912)
Within this program, the spelling "Dalmatian" seems to be used consistently.

Reply:
Thanks a lot for the cross checks.

Someone needs to rename the demo package then, as i have no clue how to do it :-)